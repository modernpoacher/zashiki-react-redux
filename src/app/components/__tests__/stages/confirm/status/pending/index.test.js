import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'

import Pending from '@modernpoacher/zashiki-react-redux/app/components/stages/confirm/status/pending'

Enzyme.configure({ adapter: new Adapter() })

describe('@modernpoacher/zashiki-react-redux/app/components/stages/confirm/status/pending', () => {
  it('renders', () => {
    const component = (
      <Pending />
    )

    expect(toJson(shallow(component)))
      .toMatchSnapshot()
  })
})
