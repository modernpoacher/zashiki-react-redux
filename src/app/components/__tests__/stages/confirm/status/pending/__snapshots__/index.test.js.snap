// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@modernpoacher/zashiki-react-redux/app/components/stages/confirm/status/pending renders 1`] = `
<div
  className="confirm pending"
/>
`;
